File: src/main/java/com/lawencon/elearn/dto/exammaterial/ExamMaterialInsertResDto.java
package com.lawencon.elearn.dto.exammaterial;

public class ExamMaterialInsertResDto {
	private ExamMaterialInsertDataResDto data;
	private String message;
	public ExamMaterialInsertDataResDto getData() {
		return data;
	}
	public void setData(ExamMaterialInsertDataResDto data) {
		this.data = data;
	}
	public String getMessage() {
		return message;
	}
	public void setMessage(String message) {
		this.message = message;
	}
}

File: frontend/src/app/dto/exam-material/exam-material.insert.res.dto.ts
import { ExamMaterialInsertDataResDto } from "./exam-material.insert.data.res.dto";

export class ExamMaterialInsertResDto{
    private _data! : ExamMaterialInsertDataResDto
	private _message! : string

    public get data(): ExamMaterialInsertDataResDto {
        return this._data;
    }

    public set data(data: ExamMaterialInsertDataResDto) {
        this._data = data;
    }

    public get message(): string {
        return this._message;
    }

    public set message(message: string) {
        this._message = message;
    }

}